src/traces/bar/attributes.js

```diff
@@ -42,7 +42,16 @@ var marker = extendFlat({
         editType: 'style',
         description: 'Sets the opacity of the bars.'
     },
-    pattern: pattern
+    pattern: pattern,
+    cornerradius: {
+        valType: 'any',
+        editType: 'plot',
+        description: [
+            'Sets the rounding of corners. May be an integer number of pixels,',
+            'or a percentage of bar width(as a string). Defaults to `layout.barcornerradius` if set; ',
+            'otherwise 0.'
```

Comment:
I think "Defaults to `layout.barcornerradius" is enough and this shouldn't be set to zero otherwise i.e. to handle the case of stacked bars. Am I missing something?

Reply:
Sounds find to me -- I've removed the 0 default for both properties.